src/pages/index.js

```diff
@@ -1,91 +1,180 @@
 import React from 'react';
-import { Redirect } from 'react-router';
 import clsx from 'clsx';
 import Layout from '@theme/Layout';
 import Link from '@docusaurus/Link';
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 import styles from './styles.module.css';
+import { ResponsivePlayer } from '../components'
 
 const features = [
   {
-    title: <>Easy to Use</>,
-    imageUrl: 'img/undraw_docusaurus_mountain.svg',
+    title: <>Server management</>,
+    imageUrl: 'img/server.png',
     description: (
       <>
-        Docusaurus was designed from the ground up to be easily installed and
-        used to get your website up and running quickly.
+        Deploy and manage your own instance of the Temporal Server.
       </>
     ),
+    goto: '/docs/server-introduction',
   },
   {
-    title: <>Focus on What Matters</>,
-    imageUrl: 'img/undraw_docusaurus_tree.svg',
+    title: <>Application development</>,
+    imageUrl: 'img/layers.png',
     description: (
       <>
-        Docusaurus lets you focus on your docs, and we&apos;ll do the chores. Go
-        ahead and move your docs into the <code>docs</code> directory.
+        Start building invincible applications using an SDK in the language of your choice!
       </>
     ),
+    goto: '/application-development',
   },
   {
-    title: <>Powered by React</>,
-    imageUrl: 'img/undraw_docusaurus_react.svg',
+    title: <>System tools</>,
+    imageUrl: 'img/settings.png',
     description: (
       <>
-        Extend or customize your website layout by reusing React. Docusaurus can
-        be extended while reusing the same header and footer.
+        Use our tooling to visualize and manage different aspects of your system.
```

Comment:
suggestion (take it or leave it) - again, specific mentions of product so they dont have to click thru to get it

```suggestion
        How to use Temporal Web and <code>tctl</code> to debug and manage Temporal.
```